### DISCUSSION TITLE
The knowledge base cannot be saved

### DISCUSSION THREAD

**Discussion by copyyour** [YetiForceCompany/YetiForceCRM#15169] [General]
Help, I added content to the knowledge base and clicked save. It also shows that the storage is successful, but when I open the knowledge base page, nothing is saved. I don't know what's going on. I ask for help. Thanks!

**Comment by mariuszkrzaczkowski** [comment]
![119500861-8d4fd580-bd68-11eb-84e0-9a381e3e002d](https://user-images.githubusercontent.com/10198654/132998086-6de4d44f-d210-49bf-988b-fc231a6b6677.jpg)
